src/cmds/codegen/codegen.ts

```diff
@@ -128,27 +127,41 @@ export class Codegen {
         }
 
         if (generator === 'typegen') {
+
+          if (!output.typings || output.typings === '') {
+            throw new Error("Please provide output.typings path to use typegen")
```

Comment:
Can we make this error message actionable? How can I usually fix this problem?

Reply:
`Please provide output.typings path in graphql config to use typegen` - does this sound better? I basically want to notify the user that they need to provide the `typings` option in the `codegen` extension.